src/api/authentication.js

```diff
@@ -217,5 +225,42 @@ export async function authenticate (ctx, next) {
   return next()
 }
 
+export async function getEnabledAuthenticationTypes (ctx, next) {
+  // Log level for messages logged
+  let logLevel = 'error'
+
+  try {
+    if (!authorisation.inGroup('admin', ctx.authenticated)) {
+      ctx.statusCode = 403
+      const error = `User ${ctx.authenticated.email} is not an admin, API access to get enabled authentication types denied.`
+      logLevel = 'info'
+      throw Error(error)
+    }
+
+    if (
+      !config.authentication ||
+      !Object.keys(config.authentication).length
+    ) {
+      throw Error('Invalid authentication Types, openhim not configured correctly')
```

Comment:
The wording here isn't quite right. Maybe something like below

```suggestion
      throw Error('No authentication enabled, invalid OpenHIM configuration')
```